bedrock/basic/send_message.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+import sys
+
+from temporalio.client import Client
+from workflows import BasicBedrockWorkflow
+
+
+async def main(prompt):
```

Comment:
```suggestion
async def main(prompt: str) -> str:
```

Usually best to type-hint things to/from workflow